test/test_util.rb

```diff
@@ -1,26 +0,0 @@
-require 'minitest/autorun'
-
-require 'beaker-hostgenerator/util'
-
-class TestUtil < Minitest::Test
```

Comment:
Since I got rid of the Utils class, this test class made less sense, and since the behavior it's testing is redundant with all the code paths exercised by the test fixtures, I figured it didn't need to be kept around.